Break the usage guide page into smaller pages (#23)
wayfair-incubator/columbo: docs/usage-guide/fundamentals.md

# Usage Guide

> This is section provides detailed descriptions of al the ways `columbo` can be used. If you are new to `columbo`, the

Reviewer: ```suggestion
This section provides detailed descriptions of all the ways `columbo` can be used. If you are new to `columbo`, the
```
